packages/discord.js/src/managers/GuildMemberManager.js

```diff
@@ -441,6 +441,32 @@ class GuildMemberManager extends CachedManager {
     return this.guild.bans.remove(user, reason);
   }
 
+  /**
+   * Adds a role to a member.
+   * @param {GuildMemberResolvable} user The member to add a role to
+   * @param {RoleResolvable} role The role to add
+   * @param {string} [reason] The reason for adding the role
+   * @returns {Promise<void>}
+   */
+  async addRole(user, role, reason) {
+    const userId = this.guild.members.resolveId(user);
+    const roleId = this.guild.roles.resolveId(role);
+    await this.client.rest.put(Routes.guildMemberRole(this.guild.id, userId, roleId), { reason });
+  }
```

Comment:
Other member manage methods return `Promise<GuildMember|User|Snowflake>`.
These methods should also do this.

Reply:
I was thinking returning the role might be a good idea, but this seems better